Add Innovation Specialist position description (PD) – 13

pixelwake opened the pull request:
First draft of what we could share about the Innovation Specialist PD.

Commit:
Created file for Inno. Spec. PD - 13

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,88 @@
+##Innovation Specialist, GS-0301-13
```
@rooneywp To ensure that this position gets added and displayed correctly, please add the lines that are at the start of the other position descriptions (ex: lines 1 through 7 of https://github.com/18F/joining-18f/edit/18f-pages/_positions/content-designer.md). If you'd like to pair on that, let me know.

fureigh replied:
Go @rooneywp go!

![rocket-corgi](https://cloud.githubusercontent.com/assets/1244599/11603559/717a5c12-9a97-11e5-9e81-2878b82196ad.gif)

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,88 @@
+##Innovation Specialist, GS-0301-13
+This position is located in the General Services Administration (GSA), Office of
```
Looks like a bunch of line breaks came through in copy-and-paste. Please remove the ones that aren't meant to demarcate ends of paragraphs.

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,88 @@
+policy, and programmatic guidance to government. The position involves
+access to PII data.
+
+###Major duties:
```
Should the names of the headings be consistent across position descriptions? Asking because these don't currently match the ones in, for instance, the [Content Designer](https://github.com/18F/joining-18f/blob/18f-pages/_positions/content-designer.md) position description.

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,88 @@
+- Empathy and Emotional Intelligence: Experience in clientfacing
+roles, navigating bureaucracy, business development, or delivering
+services with a strong userfacing
+or consumeroriented
```
Something about the copy-and-paste is losing hyphens and replacing them with a different character that's only visible when editing this markdown. To avoid the problem in the future I recommend copy-and-pasting into a plain text editor (i.e., one that doesn't add formatting) and copy-and-pasting again from there.

pixelwake replied:
@fureigh :+1: I'll correct this

pixelwake commented:
As mentioned in #127, progress is pending while I research policy

Commit:
Formatting changes: Returns, added lines 1-6

pixelwake commented:
Updated the formatting. Still pending content decisions based on policy requirements.

@fureigh re: standard format. The Innovation Specialist files will not necessarily match the format of the performance profiles since we are really pulling information directly from the PD.

fureigh commented:
@rooneywp Is this ready for review? Sincere question, asking b/c it's assigned to you and I don't want to interrupt if it's a work in progress.

pixelwake commented:
@fureigh Good question :) The current content of the file is ready for review but there will need to be more changes/additions before the file could go public. (best practice question inc) So would we merge this version knowing that it is "incomplete"? My guess would be yes, and there should be an issue that addresses the over all need and captures all the steps to complete it (this being one of them)?

fureigh commented:
@rooneywp Let's definitely not merge something whose content hasn't been signed off on. I'll update the title of this issue to indicate that it's a work in progress.

Commit:
Update Innovation-Specialist-GS-13.md

Commit:
Update Innovation-Specialist-GS-13.md

Commit:
Update Innovation-Specialist-GS-13.md

Commit:
Update Innovation-Specialist-GS-13.md

pixelwake commented:
@fureigh This is ready for review

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,38 @@
+---
+title: Innovation Specialist, GS-0301-13
+permalink: who-we-are-hiring/positions/Innovation-Specialist-GS-13/
```
URLs are generally case-sensitive, so let's keep the permalinks all-lowercase.

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,38 @@
+---
+title: Innovation Specialist, GS-0301-13
+permalink: who-we-are-hiring/positions/Innovation-Specialist-GS-13/
+team: 18F
```
The [content designer example](https://github.com/18F/joining-18f/edit/18f-pages/_positions/content-designer.md) has the value `design` instead of `18F`. Does it still make sense to talk about teams? If so, how can we do so consistently?

pixelwake replied:
Maybe we don't need this field at all? Technically, this PD doesn't belong to anyone team. I only left it in case there was some other function that used it for something.

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,38 @@
+##Innovation Specialist, GS-0301-13
+**GS Grade Level:** 13
+
+This position is located in the General Services Administration (GSA), Office of Citizen Services Innovation and Technology & 18F (OCSIT/18 F). The 18F office is charged with developing innovative digital tools and services. 18F uses lean startup and agile development principles to make easy things easy and hard things possible. As an Innovation Specialist, the incumbent serves the government and ...
```
"(OCSIT/18 F)" --> "(OCSIT/18F)", yes?

fureigh committed:
Various copy edits

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,39 @@
+- Skill in analysis in order to apply a wide range of qualitative and/or quantitative methods for assessing and improving program effectiveness
+
+Candidates are also evaluated for their competencies in key areas, including:
+- Analysis and Creative Problem Solving: Expertise determining user needs or identifying market opportunities and designing and deploying solutions
```
@rooneywp What's the rationale behind capitalizing these? If they're just meant to be easily browsable, I suggest making them bold instead (ex: **Technology acumen:**).

fureigh commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,39 @@
+- Technology Acumen: Knowledge of standard and emergent web technologies
+- Entrepreneurship and Product Management: experience delivering products or services from concept to market
+- Approach and Methodology: Expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and Emotional Intelligence: Experience in clientfacing roles, navigating bureaucracy, business development, or delivering services with a strong userfacing or consumeroriented element
```
The case of the disappearing hyphens! Copy-paste issue again?

Commit:
Minor formatting edits

Commit:
minor copy edit

pixelwake commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,39 @@
+- Entrepreneurship and product management: experience delivering products or services from concept to market
+- Approach and methodology: expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and emotional intelligence: experience in clientfacing roles, navigating bureaucracy, business development, or delivering services with a strong userfacing or consumer oriented element
+- Tools and execution: experience in design, development and deployment of solutions leveraging current and emergent technologies and best practices
```
Removed the capitalization from this section. Went through and deleted and re added the hyphens, hopefully this solved the issue.

Commit:
minor formatting edit

pixelwake commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,39 @@
+- Entrepreneurship and product management: Experience delivering products or services from concept to market
+- Approach and methodology: Expertise applying frameworks that emphasize delivery, iteration, quantitative and qualitative learnings, and responsiveness
+- Empathy and emotional intelligence: Experience in clientfacing roles, navigating bureaucracy, business development, or delivering services with a strong userfacing or consumer oriented element
+- Tools and execution: Experience in design, development and deployment of solutions leveraging current and emergent technologies and best practices
```
Fixed cases.

pixelwake commented on _positions/Innovation-Specialist-GS-13.md:
```diff
@@ -0,0 +1,39 @@
+---
+title: Innovation Specialist, GS-0301-13
+permalink: who-we-are-hiring/positions/innovation-specialist-gs-13/
+team: 18F
```
Fixed case in permalink.

Commit:
minor formatting edits

Commit:
Copy edits

Didn't touch the structure — just made a few minor punctuation and spelling updates. This looks good to go!

Commit:
Merge pull request #171 from 18F/kategarklavs-patch-1

Copy edits

pixelwake commented:
Final changes made, merging.
